tox.ini

```diff
@@ -15,6 +15,18 @@ deps =
     flake8==2.4.0
 commands=flake8 redshift_sqlalchemy tests
 
+[testenv:docs]
+basepython=python
+changedir=docs
+deps=
+    -rrequirements-docs.txt
+commands=
+    sphinx-build -W -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
+
+[flake8]
+max-line-length = 89
```

Comment:
No, my apologies. That was an oversight. I'll remove it and make sure tests still pass.